### PR TITLE
Add MiKTeX and texify support in AppVeyor CI

### PR DESCRIPTION
**Pull request by yitzchak**
First get environment initialized and make sure all binaries can be found. `texify` support will be added once AppVeyor completes and succeeds with existing specs.

### CODE REVIEW

File: script/cibuild [hunk 1]
```diff
@@ -27,8 +31,12 @@ exec_ci() {
 
 main() {
   import_dependencies
-  install_texlive
   install_knitr
+  if [[ "${TEX_DIST}" == "miktex" ]]; then
```
Comment (thomasjo) [inline script/cibuild]: If we change this to `"${TEX_DIST:=texlive}"` we can remove the explicit defaults in `.travis.yml` and `.appveyor.yml`.
Unless you object, I'll tweak that while debugging the build.

Comment (yitzchak) [inline script/cibuild]: Sounds great!

**Comment by yitzchak** [comment]
It does not look the spec failure is due to MiKTeX package installation. I ran `mpm --admin --list` before and after the Rscript hack on a separate branch and did not see a difference in the list of installed packages.

**Comment by thomasjo** [comment]
This should be good to go. We **have** to squash this when we merge — I do not want these experimental commits in the commit history 😝

**Pull request merged**